util/reggen/field.py

```diff
@@ -328,3 +377,30 @@ def sw_readable(self) -> bool:
 
     def sw_writable(self) -> bool:
         return self.swaccess.key != 'ro'
+
+    def apply_alias(self, alias_field: object, where: str) -> None:
```

Comment:
Any reason that `alias_field` isn't of type `Field` here? (You can use quotes like this: `alias_field: 'Field'` to allow use of the type before the end of the class). That way, you could get rid of the `isinstance` check on line 388.

Reply:
Oh that is interesting - I did not know that you could do that and thought this was the recommended way... let me try and fix that.